guides/common/modules/proc_installing-ovirt-plugin.adoc

```diff
@@ -0,0 +1,16 @@
+[id="installing-the-oVirt-plug-in_{context}"]
+= Installing the {oVirt} plug-in
+
+Install the {oVirt} plug-in to attach an {oVirt} compute resource provider to {Project}.
+This allows you to manage and deploy hosts to {oVirt}.
```

Comment:
Very small nit-pick. I was taught to avoid anthropomorphism ("the plug-in allows you to...."). It's your call. If all the other plug-in procedures are worded the same way, no need to update them all.

Side note: I thought oVirt compute resources were going to be deprecated.